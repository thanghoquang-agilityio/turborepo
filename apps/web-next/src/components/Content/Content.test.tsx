import { render } from '@testing-library/react'

import { Content } from '.'

describe('Content', () => {
  it('Renders empty title and subtitle match snapshot', () => {
    const { container } = render(<Content />)
    expect(container).toMatchSnapshot()
  })

  it('Renders have title and subtitle match snapshot', () => {
    const { container } = render(<Content title='Title' subTitle='subTitle' />)
    expect(container).toMatchSnapshot()
  })
})
