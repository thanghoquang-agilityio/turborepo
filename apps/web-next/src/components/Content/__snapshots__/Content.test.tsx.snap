// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content Renders empty title and subtitle match snapshot 1`] = `
<div>
  <div
    class="flex flex-col h-full gap-4 items-center justify-center"
  >
    <p
      class="  no-underline text-ellipsis whitespace-nowrap overflow-hidden text-primary-300 font-bold md:text-5xl text-3xl md:leading-10 leading-[28px]"
    >
      COMING SOON
    </p>
    <p
      class="text-primary-200 dark:text-primary-300 font-semibold text-lg no-underline text-ellipsis whitespace-nowrap overflow-hidden "
    >
      Feature is under development
    </p>
    <a
      class="mt-3"
      href="/"
    >
      <button
        aria-label="Button"
        class="z-0 group relative inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap subpixel-antialiased overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 px-4 [&>svg]:max-w-[theme(spacing.8)] transition-transform-colors-opacity motion-reduce:transition-none data-[hover=true]:opacity-hover gap-2.5 font-semibold bg-primary-300 text-background-100 sm:py-4 sm:min-w-[136px] sm:h-12 sm:text-base sm:rounded-[50px] py-2 min-w-20 h-9 text-sm rounded-[30px]"
        type="button"
      >
        Go Home
      </button>
    </a>
  </div>
</div>
`;

exports[`Content Renders have title and subtitle match snapshot 1`] = `
<div>
  <div
    class="flex flex-col h-full gap-4 items-center justify-center"
  >
    <p
      class="  no-underline text-ellipsis whitespace-nowrap overflow-hidden text-primary-300 font-bold md:text-5xl text-3xl md:leading-10 leading-[28px]"
    >
      Title
    </p>
    <p
      class="text-primary-200 dark:text-primary-300 font-semibold text-lg no-underline text-ellipsis whitespace-nowrap overflow-hidden "
    >
      subTitle
    </p>
    <a
      class="mt-3"
      href="/"
    >
      <button
        aria-label="Button"
        class="z-0 group relative inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap subpixel-antialiased overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 px-4 [&>svg]:max-w-[theme(spacing.8)] transition-transform-colors-opacity motion-reduce:transition-none data-[hover=true]:opacity-hover gap-2.5 font-semibold bg-primary-300 text-background-100 sm:py-4 sm:min-w-[136px] sm:h-12 sm:text-base sm:rounded-[50px] py-2 min-w-20 h-9 text-sm rounded-[30px]"
        type="button"
      >
        Go Home
      </button>
    </a>
  </div>
</div>
`;
